presto-benchmark-runner/src/main/java/com/facebook/presto/benchmark/event/BenchmarkSuiteEvent.java

```diff
@@ -0,0 +1,73 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.benchmark.event;
+
+import com.facebook.airlift.event.client.EventField;
+import com.facebook.airlift.event.client.EventType;
+
+import javax.annotation.concurrent.Immutable;
+
+import java.util.Optional;
+
+import static com.facebook.presto.benchmark.event.BenchmarkPhaseEvent.CompletionStatus;
+import static com.facebook.presto.benchmark.event.BenchmarkPhaseEvent.CompletionStatus.FAILED;
+import static com.facebook.presto.benchmark.event.BenchmarkPhaseEvent.CompletionStatus.SUCCEEDED;
```

Comment:
There might be some confusion here. We should either
- Move `CompletionStatus` to top-level, and use it in both `BenchmarkPhaseEvent` and `BenchmarkSuiteEvent`
- Or use separate `Status` class that are nested inside each event class.

I would prefer the 2nd option, since we'll be able to have different status values for phase event and suite event.